### ISSUE TITLE
[FR] Support search functionality in mobile

### ISSUE THREAD

**Issue by Vivian-appflowy** [AppFlowy-IO/AppFlowy#6604]
### Description

Based on community feedback, users would like the ability to search for any term in the mobile app, similar to the existing functionality in the desktop app. The search should return all matching results across the entire workspace.

### Impact

Reduce the time users spend finding specific notes, especially for those with a large number of pages.

**Comment by annieappflowy** [comment]
Hi there everyone.  Just started using AppFlowy self host and it’s great.  I’ve been looking for a Notion alternative for home.  I have a couple of issues with the app though:  search is not very good. I’d say it’s not even usable right now.  You also can search with tables.  Mobile doesn’t even have search yet.

**Comment by philiprenich** [comment]
Yes please! The feature is there in the code, yes? Just needs to be designed in to the mobile side of the codebase? Definitely necessary as it is harder to navigate large workspaces on a mobile device than a large desktop UI.
